src/git.rs

```diff
@@ -398,11 +398,11 @@ pub fn squash_index(env: &Environment) -> Result<(), PerformError> {
     let now = Utc::now().format("%Y-%m-%d");
     let original_head = repo.head_oid()?.to_string();
     let msg = format!("Collapse index into one commit\n\n\
-
+        \n\
         Previous HEAD was {}, now on the `snapshot-{}` branch\n\n\
-
+        \n\
         More information about this change can be found [online] and on [this issue].\n\n\
-
+        \n\
```

Comment:
We could just remove these lines instead of inserting `\n`, couldn't we? This adds one more blank line so the total number will be 2. I think 1 is fine here.

Reply:
Oh, oops!  I didn't notice there were already two newlines.  I have pushed an update that just removes the blank lines. It's maybe not as clear how it is formatted, though I'm not sure it really matters.